sdk/sdk/OBAApplication.h

```diff
@@ -0,0 +1,45 @@
+//
+//  OBAApplication.h
+//  OneBusAwaySDK
+//
+//  Created by Dima Belov on 4/25/15.
+//  Copyright (c) 2015 One Bus Away. All rights reserved.
```

Comment:
If it includes others code technically it should include all authors and
the year range:
´Copyright (c) 2011-2015 Brian Ferris, x, y, ..., Dima Belov´

I'd do the best you can to assemble this given the Git history.
On May 26, 2015 10:54 PM, "Dima Belov" <notifications@github.com> wrote:

> In sdk/sdk/OBAApplication.h
> <https://github.com/OneBusAway/onebusaway-iphone/pull/425#discussion_r31099657>
> :
>
> > @@ -0,0 +1,45 @@
> > +//
> > +//  OBAApplication.h
> > +//  OneBusAwaySDK
> > +//
> > +//  Created by Dima Belov on 4/25/15.
> > +//  Copyright (c) 2015 One Bus Away. All rights reserved.
>
> I didn't actually write like 99% of the code, just extracted it, still
> applies?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OneBusAway/onebusaway-iphone/pull/425/files#r31099657>
> .
>